crates/turbopack-core/src/issue/mod.rs

```diff
@@ -426,15 +426,15 @@ impl CapturedIssues {
 #[turbo_tasks::value]
 #[derive(Clone)]
 pub struct IssueSource {
-    pub asset: AssetVc,
+    pub source: AssetVc,
     pub start: SourcePos,
     pub end: SourcePos,
 }
 
 #[turbo_tasks::value_impl]
 impl IssueSourceVc {
     #[turbo_tasks::function]
-    pub async fn from_byte_offset(asset: AssetVc, start: usize, end: usize) -> Result<Self> {
+    pub async fn from_byte_offset(source: AssetVc, start: usize, end: usize) -> Result<Self> {
```

Comment:
I wasn't able to change this to `SourceVc` yet, since it's called also called `ModuleVc`